guides/callbacks/callbacks.md

```diff
@@ -0,0 +1,22 @@
+# Bandwidth Callbacks
+
+## Bandwidth Callbacks Overview
+
+Bandwidth's APIs operate on a system of callbacks. Most API requests send callbacks, and notices of incoming events (phone call, text message) are sent via callbacks as well.
+
+## How To Utilize Callbacks
+In order to utilize callbacks, you must run a server with a URL that is expecting to receive callbacks from Bandwidth. This URL must be provided to Bandwidth through your Bandwidth application settings, or as part of your API request. For voice callbacks, this endpoint must return valid BXML for Bandwidth to use. More information on Bandwidth applications can be found [here](../../applications/about.md).
```

Comment:
try to avoid linking like [here] and instead link the relevant word: https://www.w3.org/QA/Tips/noClickHere

Reply:
Fixed all occurrences of [here] 👍